### PR TITLE
ff 0.11, group 0.11, etc.

### PR DESCRIPTION
**Pull request by str4d**

### CODE REVIEW

File: zcash_client_backend/CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ and this library adheres to Rust's notion of
 ## [Unreleased]
 ### Changed
 - MSRV is now 1.51.0.
+- Bumped dependencies to `ff 0.11`, `group 0.11`, `bls12_381 0.6`, `jubjub 0.8`.
```
Comment (daira) [inline zcash_client_backend/CHANGELOG.md:11]:
```suggestion
- Bumped dependencies to `ff 0.11`, `group 0.11`, `bls12_381 0.6`, `jubjub 0.8`,
  and the most recent unreleased versions of `halo2` and `orchard`.
```

File: zcash_primitives/src/sapling.rs [hunk 2]
```diff
@@ -216,7 +217,7 @@ impl ViewingKey {
         // Drop the most significant five bits, so it can be interpreted as a scalar.
         h[31] &= 0b0000_0111;
 
-        SaplingIvk(jubjub::Fr::from_repr(h).expect("should be a valid scalar"))
+        SaplingIvk(jubjub::Fr::from_repr(h).unwrap())
```
Comment (ebfull) [inline zcash_primitives/src/sapling.rs:220]: We should eventually add an `expect` method to `CtOption`.

**Review by ebfull — APPROVED** [review]

File: zcash_primitives/CHANGELOG.md [hunk 3]
```diff
@@ -54,6 +54,7 @@ and this library adheres to Rust's notion of
 
 ### Changed
 - MSRV is now 1.51.0.
+- Bumped dependencies to `ff 0.11`, `group 0.11`, `bls12_381 0.6`, `jubjub 0.8`.
```
Comment (daira) [inline zcash_primitives/CHANGELOG.md:57]:
```suggestion
- Bumped dependencies to `ff 0.11`, `group 0.11`, `bls12_381 0.6`,
  `jubjub 0.8`, `pasta_curves 0.2`
```

Comment (str4d) [reply zcash_primitives/CHANGELOG.md:57]: It's fine as-is; it's within 80 characters, which is what this time is wrapped at IIRC.

Comment (daira) [reply zcash_primitives/CHANGELOG.md:57]: Right, but `pasta_curves` was bumped.

Comment (str4d) [reply zcash_primitives/CHANGELOG.md:57]: We haven't cut a release containing Orchard logic in a public API yet, so bumping the `pasta_curves` crate doesn't change anything.

**Pull request merged**